rules/windows/defense_evasion_clearing_windows_security_logs.toml

```diff
@@ -0,0 +1,22 @@
+[metadata]
+creation_date = "2020/11/12"
+maturity = "development"
+updated_date = "2020/11/12"
+
+[rule]
+author = ["Anabella Cristaldi"]
+description = "Identifies attempts to clear Windows Security log stores"
+from = "now-9m"
+index = ["winlogbeat-*"]
+language = "kuery"
+license = "Elastic License"
+name = "Clearing Windows Security Logs"
+risk_score = 21
+rule_id = "45ac4800-840f-414c-b221-53dd36a5aaf7"
+severity = "low"
+tags = ["Windows", "Elastic"]
+type = "query"
+
+query = '''
+event.module:security and event.action:audit-log-cleared
```

Comment:
```suggestion
event.action:("audit-log-cleared" or "Log clear")
```

Reply:
Hi @threat-punter , I did not consider the event 104 because at the moment there is no winlogbeat module for System's events and therefore those events are not translated to ECS. 
@andrewkroh is it planned some module for system events? I can start a new one...
I'll be watching if in the future system's logs are normalized into ECS in order to update the rule.
Thank you!

Reply:
> is it planned some module for system events? I can start a new one...

I hadn't planned on one, but it looks like there are important events that could use ECS categorization so I think it should exist.

Reply:
@andrewkroh  Should I added to the open https://github.com/elastic/beats/pull/20684?

Reply:
No, in general smaller, targeted PRs are preferred because they are simpler to understand, review, and cherry-pick to other branches. So I would keep them separated.